core/server/api/canary/settings.js

```diff
@@ -124,6 +131,21 @@ module.exports = {
                 }
             }
 
+            if (stripeConnectIntegrationToken) {
```

Comment:
I can't see how the current stripe config is protected by permissions, so it seems like this should work the same way it currently does - @naz do you have any insight on this?

Reply:
@allouis there is no specific permission checks for member related config values. The only thing that is protected is toggling the "members" flag itself - https://github.com/TryGhost/Ghost/commit/786eaac57ecdbfcbc4c7969f3e04c1023b537927 That said, it's not what we'd want ideally. 

What I see happening is having specific permissions once stripe configuration is extracted into separate endpoint - then it would be a lot more manageable "resource-based" approach to implement. Think current approach is OK as is, but ideally would become more granular in the future.

Reply:
Okay cool - that makes total sense, thank! :relaxed: